presto-base-jdbc/src/main/java/io/prestosql/plugin/jdbc/BaseJdbcConfig.java

```diff
@@ -49,6 +51,23 @@ public BaseJdbcConfig setConnectionUrl(String connectionUrl)
         return this;
     }
 
+    @AssertTrue
+    public boolean isConnectionUrlValid()
```

Comment:
Are we sure this is really sattisfied by all databases? I would rather not break someones deployment by over-strict validation.

Reply:
![Screenshot 2020-12-13 at 14 29 55](https://user-images.githubusercontent.com/66972/102013336-b26b0180-3d4f-11eb-8dd2-299ee33058d3.png)

Per JDBC 4.3 spec: 9.4 The DriverManager Class section

Reply:
@wendigo  are you convinced all drivers adhere to JDBC specification?

Reply:
I can go through every JDBC connector that we use.

Reply:
| Driver | Prefix |
|--------|--------|
|  Mysql      |    jdbc:mysql:// jdbc:mysql:mxj:// jdbc:mysql:loadbalance:// jdbc:mysql:replication://    |
|  H2 | jdbc:h2:  |
|  PostgreSQL      | jdbc:postgresql:       |
| Hive | jdbc:hive2:// |
| Oracle | jdbc:oracle: |
| Phoenix | jdbc:phoenix: |
| SQL Server | jdbc:sqlserver:// |
| Apache Calcite | jdbc:calcite: |
| Apache dbcp2 | jdbc:apache:commons:dbcp: |